src/main/resources/scripts/simple join and leave message.sk

```diff
@@ -0,0 +1,27 @@
+#
+# A join and leave message.
+#
+
+command /set <string> message <text>:
```

Comment:
```suggestion
command /set <text> message <text>:
```

Reply:
Should be better to use string instead of text type right?

Reply:
Skript handles both but at least if you want to use one in particular, use it everywhere

Reply:
Yeah I missed to change the 2nd text with ``string``. I will fix that.